test/ref/pack_int4.cpp

```diff
@@ -115,13 +129,13 @@ TEST_CASE(pack_int4_nchw)
                               0xBA,
                               0xDC,
                               0xFE,
-                              0x10,
-                              0x32,
-                              0x54,
-                              0x76,
-                              0x98,
-                              0xBA,
-                              0xDC,
-                              0xFE};
+                              0xFF,
```

Comment:
add tests "ref" tests for "unpack" as well

Reply:
There already is a test for pack_unpack. Isn't that enough?

Reply:
Just to clarify further, there are ref tests that check for pack operations, and then followed by its unpack counterpart. And those initial and final values are compared appropriately. Thanks.